machine/AVAILABLE_DRIVER_PLUGINS.md

```diff
@@ -0,0 +1,47 @@
+<!--[metadata]>
```

Comment:
Please convert the TOML frontmatter to YAML:

- Remove the XML comments from top and bottom
- change +++ to --- at top and bottom
- change draft = true to published: false
- change = to : everywhere